src/lib/harvest.ts

```diff
@@ -1,6 +1,8 @@
+import 'whatwg-fetch';
+
 import { getAuthToken, getEnableConfig, TrelloPromise } from './store';
 
-export const API_BASE_URL = 'https://api.harvestapp.com/api/v2/';
+export const API_BASE_URL = 'https://api.harvestapp.com/v2/';
```

Comment:
@carljm I guess apparently both of these work? This now matches the docs, as far as I can tell.

Reply:
(I did functionally test this -- it works the same as far as I can tell.)